src/SFA.DAS.Data.Database/Views/DAS_Employer_Transfer_Relationship.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE VIEW [Data_Pub].[DAS_Employer_Transfer_Relationship]	AS
+	SELECT
+		[Id], 
+		[SenderAccountId],
+		[ReceiverAccountId], 
+		[RelationshipStatus], 
+		[SenderUserId], 
+		[ApproverUserId], 
+		[RejectorUserId],
+		[UpdateDateTime]
+	FROM 
+		[Data_Load].[DAS_Employer_Transfer_Relationships]
+	WHERE
+		[IsLatest] = 1
```

Comment:
After speaking with Mike, this view should return everything from the view, no need for the IsLatest condition

Reply:
Done